src/content/basics/networkpools/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Network Pools
 description: A general description of network pools as a concept, it's benefits, and some details you should be aware of.
-date: 2021-01-09
```

Comment:
Please remove the date field. This is no longer needed.

Reply:
Done!